### PR TITLE
Improve Matrix Content (mxc://) URIs in tests

### PR DESCRIPTION
**Pull request by ahmadkadri**
**:heavy_check_mark: Checklist**

- [ ] A changeset describing the change and affected packages ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#changelog-and-versioning)).
- [ ] Added or updated documentation.
- [ ] Tests for new functionality and regression tests for bug fixes.
- [ ] Screenshots or videos attached (for UI changes).
- [x] All your commits have a `Signed-off-by` line in the message ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#dco)).

### CODE REVIEW

File: packages/mui/src/components/ElementAvatar/ElementAvatar.test.tsx [hunk 1]
```diff
@@ -57,15 +57,15 @@ describe('<ElementAvatar>', () => {
...
+        avatarUrl={'mxc://example.com/imageAlice'}
       />,
     );
 
     await expect(
       screen.findByRole('presentation', { hidden: true }),
     ).resolves.toHaveAttribute(
       'src',
-      expect.stringMatching(/\/_matrix\/media\/r0\/thumbnail\/alice/i),
+      expect.stringMatching(/\/_matrix\/media\/r0\/thumbnail\/.*\/imageAlice/i),
```
Comment (weeman1337) [inline packages/mui/src/components/ElementAvatar/ElementAvatar.test.tsx:68]: This should also contain the server name https://spec.matrix.org/v1.12/client-server-api/#get_matrixclientv1mediathumbnailservernamemediaid

**Review by weeman1337 — CHANGES_REQUESTED** [review]
Only one test to go. Looks good otherwise :+1:

File: packages/mui/src/components/ElementAvatar/ElementAvatar.test.tsx [hunk 2]
```diff
@@ -57,15 +57,17 @@ describe('<ElementAvatar>', () => {
...
       />,
     );
 
     await expect(
       screen.findByRole('presentation', { hidden: true }),
     ).resolves.toHaveAttribute(
       'src',
-      expect.stringMatching(/\/_matrix\/media\/r0\/thumbnail\/alice/i),
+      expect.stringMatching(
+        /https:\/\/.*\/_matrix\/media\/r0\/thumbnail\/example\.com\/.*imageAlice/i,
```
Comment (HarHarLinks) [inline packages/mui/src/components/ElementAvatar/ElementAvatar.test.tsx:69]: > it('should render avatar url'

I'm not sure, should it? Can't avatars be authenticated media? Did we miss this before when implementing auth media?

Comment (mgcm) [reply packages/mui/src/components/ElementAvatar/ElementAvatar.test.tsx:69]: I guess [this method](https://github.com/nordeck/matrix-widget-toolkit/blob/main/packages/mui/src/components/ElementAvatar/createAvatarUrl.ts) should be fully reviewed under that light.

Maybe something to work on another PR / ticket?

Comment (ahmadkadri) [reply packages/mui/src/components/ElementAvatar/ElementAvatar.test.tsx:69]: this PR only to fixed the MXC URLs in the  tests and not the implementation. If necessary,  if necessary we should open a ticket.

Comment (mgcm) [reply packages/mui/src/components/ElementAvatar/ElementAvatar.test.tsx:69]: I've opened it internally. We'll tackle this comment there.

**Review by mgcm — APPROVED** [review]

File: packages/mui/src/components/ElementAvatar/ElementAvatar.test.tsx (hunk 2 above)
Comment (HarHarLinks) [inline packages/mui/src/components/ElementAvatar/ElementAvatar.test.tsx:69]: the URI is still incorrect, even if we keep to the r0 version. please check with the spec: https://spec.matrix.org/latest/client-server-api/#get_matrixmediav3thumbnailservernamemediaid

Comment (weeman1337) [reply packages/mui/src/components/ElementAvatar/ElementAvatar.test.tsx:69]: Also `.*` is a regexp wild card (match any char). Please remove it.

**Review by weeman1337 — APPROVED** [review]

**Pull request merged**